airflow/models/__init__.py

```diff
@@ -4263,6 +4263,14 @@ def setdefault(cls, key, default, deserialize_json=False):
         else:
             return obj
 
+    @classmethod
+    @provide_session
+    def try_get(cls, key, deserialize_json=False, session=None):
+        try:
+            return cls.get(key, deserialize_json=deserialize_json, session=session)
+        except KeyError:
+            return None
```

Comment:
I'd like to do this without an extra method. What I propose is:

```
    NO_DEFAULT_SENTINEL = object()

     def get(cls, key, default_var=NO_DEFAULT_SENTINEL, deserialize_json=False, session=None):
        obj = session.query(cls).filter(cls.key == key).first()
        if obj is None:
            if default_var is not NO_DEFAULT_SENTINEL:
                return default_var
            ...
```

Reply:
👍 I like this. However this will change the current behaviour when `default_var` is set to `None` explicitly, but maybe that is not something people do? 🤷‍♂️

Reply:
I _think_ it's probably a rare-enough case that it won't affect many people.

Still, we should add a note to UPDATING.md saying something along the lines of "if you want the old behaviour of raising when  the variable is not found, ensure `default_var` is not passed"